lib/index.js

```diff
@@ -98,6 +99,11 @@ module.exports = function (gulp, options) {
 
   styleguide(gulp, lintOptions);
 
+  gulp.task('shrinkwrap', 'Shrinkwrap the existing node modules.' +
```

Comment:
added to README and set shrinkwrapping the existing modules to the default behavior